nodejs/week1/avg.js

```diff
@@ -0,0 +1,25 @@
+const getNumbers = (process.argv[2]);
+if (getNumbers === undefined){
+    console.log("Enter numbers")
+    return
```

Comment:
empty return here seems to be irrelevant. May be best to remove it

Reply:
Hi Gaurav,
Thanks for reviewing my homework. I have made the suggested change. Please approve the merge request.